subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -491,36 +491,6 @@ static void prov_node_added(uint16_t net_idx, uint8_t uuid[16], uint16_t addr,
 	bt_mesh_shell_target_ctx.dst = addr;
 }
 
-#if defined(CONFIG_BT_MESH_PROVISIONER)
```

Comment:
I can show you in another channel

Reply:
Discussed IRL. There is another approach in shell to provision devices with OOB. However, it would very helpful if the provisioner suggested\used only OOB methods that the device provides in the capability message.